sdk/servicebus/service-bus/src/receiver.ts

```diff
@@ -174,7 +185,7 @@ export class Receiver {
     this._throwIfReceiverOrConnectionClosed();
     this._throwIfAlreadyReceiving();
 
-    if (!this._context.batchingReceiver || !this._context.batchingReceiver.isOpen()) {
+    if (!this._context.batchingReceiver) {
```

Comment:
What is driving this change?

Reply:
Looking at it it seemed unnecessary - we can call _init() multiple times on a batchingreceiver and each time will work the same (or early exit if it's already open).

This just made it consistent with the check I was doing for streamingReceiver.

(I can bring it back - I have no strong feelings on it).

Reply:
On thinking about this more, I think that the way I have it now is just simpler to reason through. We don't have to worry about any overlap/concurrency issues for the field anymore. It's either set or not and any other calls that manipulate it's state are protected by the lock.

If we don't do that then I have to start reasoning about whether it's possible for two concurrent instances of _context.batchingReceiver can be there (ie, we've swapped out an older one for a newer one and we're somehow initializing the older one). So I think this is worth keeping.